File: C/Cap05/c05ex07.c
// c05ex07.c

#include <stdio.h>

int main(void) {
  int I, N, R;

  I = 1;
  do {
    printf("Entre um valor numerico: ");
    scanf("%d", &N);
    while (getchar() != '\n'); 

    R = N * 3;

    printf("Resultado = %d\n\n", R);

    ++I;
  } while (I <= 5);

  printf("\n");
  printf("Tecle <Enter> para encerrar... ");
  while (getchar() != '\n'); 

  return 0;
}

File: JAVASCRIPT/Cap05/c05ex07.js
// c05ex07.js

const rls = require('readline-sync');

let I = 1;
do {
  let N = rls.questionInt('Entre um valor numerico: ');	

  let R = N * 3;

  console.log("Resultado = " + R + "\n");

  I++;
} while (I <= 5);

console.log("");
process.stdout.write('Tecle <Enter> para encerrar... ');
rls.question();
